### ISSUE TITLE
use `Aegir` for simulating files

### ISSUE THREAD

**Issue by Shoram444** [Shoram444/SNSensitivityEstimate#24]
Need to create own class similar to:

https://github.com/SuperNEMO-DBD/Falaise/blob/develop/plugins/examples/ExAegir/exaegir/dummy_event_generator2.h

Figure out how to plug in 
- `double kineticEnergy = _rejection_method_.shoot(grab_external_random());` : here energy would be sampled from Rasto's data
- `double cosTheta = -1.0 + 2.0 * grab_external_random().uniform();`: here angle would be sampled

- how best to integrat summation over spectrum components? ($\xi_{31}$ ?)

Basically this is important:
`void dummy_event_generator2::_load_next(::genbb::primary_event & event_,
					 bool compute_classification_)`
